lib/markdown2.py

```diff
@@ -1088,7 +1085,7 @@ def _strict_tag_block_sub(
 
         for chunk in text.splitlines(True):
             is_markup = re.match(
-                r'^(\s{0,%s})(?:</code>(?=</pre>))?(</?(%s)\b>?)' % ('' if allow_indent else '0', current_tag), chunk
+                r'^(\s{{0,{}}})(?:</code>(?=</pre>))?(</?({})\b>?)'.format('' if allow_indent else '0', current_tag), chunk
```

Comment:
What's up with the double wrap curlies here, isn't immediately clear to me

Reply:
Ah, double curly escape in the regex?

Reply:
Yes, or rather escaping format strings:

> Format strings contain “replacement fields” surrounded by curly braces `{}`. Anything that is not contained in braces is considered literal text, which is copied unchanged to the output. If you need to include a brace character in the literal text, it can be escaped by doubling: `{{` and `}}`.


https://docs.python.org/3/library/string.html#format-string-syntax